pyproject.toml

```
@@ -10,7 +10,11 @@ name = "gridfinityworkbench"
 version = "0.9.4"
 
 [project.optional-dependencies]
-dev = ["ruff", "freecad-stubs"]
+dev = ["mypy", "ruff", "freecad-stubs"]
```

Te be on par with the issue solved in #75

```suggestion
dev = ["mypy==1.15.0", "ruff", "freecad-stubs"]
```